File: fire_test.js
const nodemailer = require('nodemailer');

const transporter = nodemailer.createTransport({
    host: 'smtp.gmail.com',
    port: 587,
    secure: false, // true for 465, false for other ports
    auth: {
        user: 'misagh754@gmail.com',
        pass: 'lqqnoolxcahfcdir'  // Render ENV 'SMTP_PASS' equivalent
    }
});

const checkout_link = "https://human-probability-enginehuman.onrender.com";

const mailOptions = {
    from: '"HPE Rainmaker" <misagh754@gmail.com>',
    to: 'misagh754@gmail.com',
    subject: 'rainmaker is live',
    text: `Hey CEO,\n\nThis is an automated verification email from THE RAINMAKER.\n\nThe autonomous outreach engine is now LIVE and operational.\nDaily Limit: 40 emails/day\nPrice Point: $199/conversion\n\nThe engine will now begin processing the 342 identified startup leads.\n\n— HPE Rainmaker Engine v2.0`,
    html: `<div style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif; font-size: 14px; color: #333; line-height: 1.6;">
Hey CEO,<br><br>
This is an automated verification email from THE RAINMAKER.<br><br>
The autonomous outreach engine is now LIVE and operational.<br>
Daily Limit: 40 emails/day<br>
Price Point: $199/conversion<br><br>
The engine will now begin processing the 342 identified startup leads in Stealth Mode (7-22 min delay).<br><br>
Run your Deep Scale Analysis here ($199): <a href="${checkout_link}" style="color: #0ea5e9;">${checkout_link}</a>
<br><br>
<span style="color: #999; font-size: 11px;">Sent from HPE Analytics V1 | <a href="${checkout_link}" style="color: #0ea5e9;">humanprobability.ai</a></span>
</div>`
};

console.log("=" * 60);
console.log("IGNITING FULL FIRE MODE: Dispatching CEO Verification Email...");
console.log("=" * 60);

transporter.sendMail(mailOptions, (error, info) => {
    if (error) {
        console.log('✗ EMAIL SEND FAILED (Check App Password formatting):', error);
    } else {
        console.log('✓ FULL FIRE TEST EMAIL SENT TO INBOX:', info.response);
    }
});

File: fire_test.py
import asyncio
import os
import sys

# Add python_engine to path so imports work
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), "python_engine"))

# Inject the provided App Password
os.environ["SMTP_PASS"] = "lqqnoolxcahfcdir"
os.environ["SMTP_USER"] = "misagh754@gmail.com"

from sales.infrastructure.controller import StealthSender

async def main():
    print("="*60)
    print("IGNITING FULL FIRE MODE: Dispatching CEO Verification Email...")
    print("="*60)
    success = await StealthSender.send_email(
        subject="rainmaker is live",
        body=(
            "Hey CEO,\n\n"
            "This is an automated verification email from THE RAINMAKER.\n\n"
            "The autonomous outreach engine is now LIVE and operational.\n"
            "Daily Limit: 40 emails/day\n"
            "Price Point: $199/conversion\n\n"
            "The engine will now begin processing the 342 identified startup leads.\n\n"
            "— HPE Rainmaker Engine v2.0"
        ),
        recipient="misagh754@gmail.com",
        sender_name="HPE Rainmaker"
    )
    if success:
        print("✓ FULL FIRE TEST EMAIL SENT TO INBOX")
    else:
        print("✗ EMAIL SEND FAILED (Check App Password formatting)")

asyncio.run(main())
